### DISCUSSION TITLE
Ideal prompt template to use

### DISCUSSION THREAD

**Discussion by pranavishekhar368** [microsoft/semantic-kernel#6860] [Q&A] [answered]
I'm currently using the Handlebars template for my project, but this has been flagged as experimental for quite some time now so I'd like to know if this is something that will continue to be maintained or if there's a more stable/better template (like Prompty for instance) that I should look towards?

**Comment by sophialagerkranspandey** [comment, marked as answer]
Hello, we will move the Handlebars template out of experimental to non-experimental - we recommend you continue to use this.
